react/features/jaas/middleware.web.js

```diff
@@ -19,8 +24,35 @@ MiddlewareRegistry.register(store => next => async action => {
         if (status === STATUSES.BLOCKED) {
             store.dispatch(redirectToStaticPage('/static/planLimit.html'));
         }
+        break;
+    }
+
+    case CONFERENCE_JOINED:{
```

Comment:
style: space after `:` also, missing `break`, which can be dangerous if we add more cases to the switch. Last, please sort the case statements alphabetically.